lxqt-session/src/sessionapplication.cpp

```diff
@@ -106,10 +104,14 @@ bool SessionApplication::startup()
             });
 #endif
 
-    if (lockScreenManager->startup())
-        qDebug() << "LockScreenManager started successfully";
-    else
-        qWarning() << "LockScreenManager couldn't start";
+    if (settings.value(QLatin1String("lock_screen_before_power_actions")).toBool())
```

Comment:
I didn't do it because the other variables are not used as well, so I was thinking in a refactor lxqt-session to follow lxqt-powermanagement: a settings class with accessor methods (or public static contants).